<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

color:black;
border:1px solid grey;
background-color: rgb(255,255,255);
padding: 0 8px;
font:600  11pt;


}</string>
   </property>
   <property name="text">
    <string> LBTC STUDENTS UNION VOTING MANAGEMENT SYSTEM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>651</width>
     <height>391</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>683</y>
     <width>865</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string> Copyright @ 2025 Lusaka Business and Technical College Students Union Voting System. Developed By Fredrick Kangwa</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>99</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string> </string>
   </property>
   <property name="icon">
    <iconset theme="user-available"/>
   </property>
   <property name="iconSize">
    <size>
     <width>49</width>
     <height>49</height>
    </size>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_username_login">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_password_login">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>401</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_select_account_type">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>388</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_submit_login">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>450</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Reset_login">
   <property name="geometry">
    <rect>
     <x>505</x>
     <y>450</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
